mongodbatlas/projects.go

```diff
@@ -177,12 +181,17 @@ func (s *ProjectsServiceOp) GetOneProjectByName(ctx context.Context, projectName
 // Create creates a project.
 //
 // See more: https://docs.atlas.mongodb.com/reference/api/project-create-one/
-func (s *ProjectsServiceOp) Create(ctx context.Context, createRequest *Project) (*Project, *Response, error) {
+func (s *ProjectsServiceOp) Create(ctx context.Context, createRequest *Project, opts *CreateProjectOptions) (*Project, *Response, error) {
 	if createRequest == nil {
 		return nil, nil, NewArgError("createRequest", "cannot be nil")
 	}
 
-	req, err := s.Client.NewRequest(ctx, http.MethodPost, projectBasePath, createRequest)
+	path, err := setListOptions(projectBasePath, opts)
```

Comment:
does `setListOptions` checks if opt is nil?

Reply:
yes https://github.com/mongodb/go-client-mongodb-atlas/blob/a40fdd6c5807714665e121eff8144d942790ce49/mongodbatlas/mongodbatlas.go#L546